pkg/model/model.go

```diff
@@ -1,260 +1,86 @@
 /*
-Copyright 2021.
-
-Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
-
-    http://www.apache.org/licenses/LICENSE-2.0
-
-Unless required by applicable law or agreed to in writing, software
-distributed under the License is distributed on an "AS IS" BASIS,
-WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-See the License for the specific language governing permissions and
-limitations under the License.
+model package provides NodePackedPower, PodPowerRatio, and PodDynamicPower
+- NodePackedPower: node-level power in core, uncore, dram, and package
+- PodPowerRatio: power that each pod or system should account for when considering usage shares (ratio)
+- PodDynamicPower: power that each pod should account for when considering only its usage values
```

Comment:
This is when the hardware counter is not available, right?
I think we can make this comment clearer